catext/any-peaceful-ic.md

```diff
@@ -9,8 +9,8 @@ Any% Peaceful is exclusive (as of now) to Android due to the glitch needed to
 run the category. The current duping method is done on Android version 1.2.6.1
 (1.2.6.60) or 1.2.6.55.
 
-* The glitch used is allowed on Any% Peaceful (Item Creation) ONLY. Not on any
-other leaderboards unless explicitely stated otherwise.
-* Timing ends the frame the screen begins to fade after exiting the end.
+* The glitch used is ONLY allowed on Any% Peaceful (Item Creation). It is not
+permitted on any other leaderboards unless explicitly stated otherwise.
+* Timing ends (on the frame) when the End Poem fades in upon exiting the End.
```

Comment:
Capitalizing "End" here is kinda weird. I'm not sure if "the end" or "The End" is correct here, but I think "the End" is certainly odd

For reference the minecraft blog spells it as "The End"
https://www.minecraft.net/en-us/article/around-block--end-highlands
```suggestion
* Timing ends (on the frame) when the End Poem fades in upon exiting The End.
```

Reply:
"The" is only capitalised when it's at the start of a sentence or is part of a title.

In the blog you referrenced, there are instances of "the End" as opposed to "The End". The only time "The End" appears is at the start of a sentence or as a title. In the context: "The first, confusingly, is called **The End** – that’s the bit where you fight the Ender Dragon." it is referred to as a title.

You can see it with "The Nether" here:
https://www.minecraft.net/en-us/article/the-bedrock-nether-update-beta-here-